source/i2c_io.c

```diff
@@ -9,6 +9,7 @@
 #include "HL_i2c.h"
 #include "os_semphr.h"
 #include <stdint.h>
+#include <stdio.h>
```

Comment:
what's this used for?

Reply:
Don't remember. I think we were using printfs at some point. Removed